apps/i18n/common/en_us.json

```diff
@@ -407,6 +407,8 @@
   "childAccountConsentValidPermission": "Permission:",
   "childAccountConsentValidPermissionGranted": "Granted on {date}",
   "childAccountConsentValidMessage": "Thank you for approving your child’s Code.org account! You will receive a confirmation email within 24–48 hours. Please let your child know you’ve approved their account so they can get started learning and coding today!",
+  "childAccountPolicy_CreateSectionsWarning": "If you have students under 13, they may require parental consent to use a personal login. Please go back and use picture password, secret word, or school-managed sections to avoid any interruptions for students under 13.",
+  "childAccountPolicy_LearnMore": "Learn more about parental consent",
```

Comment:
Using a new name space Child Account Policy since these messages will appear in places other than the child account Consent pages.